### ISSUE TITLE
The Wordcount crash if the wordcount container is not defined

### ISSUE THREAD

**Issue by ma2ciek** [ckeditor/ckeditor5-word-count#16]
Right now the logic is as follows:

```js
// constructor()
this._config = editor.config.get( 'wordCount' ) || {};
```

```js
// init()
if ( isElement( this._config.container ) ) {
	this._config.container.appendChild( this.wordCountContainer );
}
```

```js
// destroy()
destroy() {
	this._outputView.element.remove();
	this._outputView.destroy();

	super.destroy();
}
```

```js
get wordCountContainer() {
	// ...

	if ( !this._outputView ) {
		this._outputView = new View();
	}

	// ...
}
```

So the `this._outputView` might be undefined in the `destroy()` method and this will end up with the `Cannot read property 'element' of undefined` error.

Ref: https://github.com/ckeditor/ckeditor5-angular/issues/123.

**Comment by ma2ciek** [comment]
Also, the `this._config.container` configuration isn't mention in the https://ckeditor.com/docs/ckeditor5/latest/features/word-count.html

**Comment by ma2ciek** [comment]
ping @msamsel.

**Comment by msamsel** [comment]
@ma2ciek I've proposed a PR fixing this issue.

**Comment by dyh333** [comment]
@msamsel hi, I want to know if this bug has fixed in  @ckeditor/ckeditor5-word-count v10.0.1? I met the same problem in angular8 when destroy ckeditor5 component?

**Comment by ma2ciek** [comment]
It isn't released yet AFAIK. The quick fix, for now, is to get the `wordCountContainer` after the editor initializes. even with a  `editor.plugins.get( 'WordCount' ).wordCountContainer;`. This will invoke the getter and the `_outputView` should be created.
